src/Main.jsx

```diff
@@ -4,59 +4,30 @@ import data from './data.json';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 import Container from 'react-bootstrap/Container';
-import { render } from 'react-dom';
 
 class Main extends React.Component {
     render() {
-        let title = "uniWhal";
+        let hornedBeastComponents = [];
+        data.forEach((beasts) => {
+            hornedBeastComponents.push(
+                <Col xs={4} md={4} lg={4}>
+                    <HornedBeasts 
+                        name={beasts.name}
```

Comment:
beasts.name is undefined, check the names of the beasts properties in data.json, make sure you're using the right property!